sympy/functions/elementary/exponential.py

```diff
@@ -1040,3 +1048,12 @@ def _eval_nseries(self, x, n, logx):
 
             return s + Order(x**n, x)
         return super(LambertW, self)._eval_nseries(x, n, logx)
+
+    def _eval_is_zero(self):
+        x = self.args[0]
+        if len(self.args) == 1:
+            k = S.Zero
```

Comment:
As a general principle I think it would be better to use properties like x and k than to manipulate args directly. We should try to limit the number of places that depend on the exact layout of args.